SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +31,129 @@ def customise(process):
 
     return process
 
+
+def customise_SimTkOnly(process):
```

Comment:
@ebrondol @makortel as I mentioned in a previous comment, I think the proper solution is to define separate cmsDriver steps with their own cff files containing only the desired modules. Forcing this into the Era system does not make sense to me. Reference:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/Applications/python/ConfigBuilder.py
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/StandardSequences/python/Sim_cff.py
etc.